raw/README.md

```diff
@@ -1,11 +1 @@
-# Source directory
-
-This directory contains the "raw" source data of the dataset from which the
-CLDF dataset in [`cldf/`](../cldf) is derived.
-
-
```

Comment:
Maybe we should keep these notes somewhere. They coud for example go into comments in the code where the lookup in the `comments` column is implemented.